portfolio/src/main/java/com/google/sps/data/Data.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+import java.util.ArrayList;
+import java.lang.String;
+
+public final class Data{
```

Comment:
What's the advantage of creating this class to hold your messages instead of converting the array directly to JSON?

Reply:
I couldn't find an example of Gson turning an array directly to JSON as all the examples required objects, so I made a class

Reply:
Have you tried just calling toJson() with messages? I googled around and it seems like it should be possible: https://sites.google.com/site/gson/gson-user-guide#TOC-Collections-Examples (Look at the Array and Collections examples.)

Note that you'll have to adjust your client-side code too. If before you were iterating over site.messages, you'll now just iterate over site directly.

(For the record, the thing I searched for was "java gson arraylist to json")